pkg/worker/worker.go

```diff
@@ -251,6 +250,24 @@ func NewWorker() (*Worker, error) {
 	}, nil
 }
 
+func (s *Worker) IsCRIUAvailable() bool {
+	if s.cedanaClient == nil {
+		return false
+	}
+
+	poolName := os.Getenv("WORKER_POOL_NAME")
```

Comment:
are we just passing the whole pool cause in the future it might be more convenient than adding another extra env var just for criu enabled?